AstroPhotometry/View/MainWindow.xaml.cs

```diff
@@ -49,7 +50,27 @@ private void Button_Click(object sender, RoutedEventArgs e)
                 photo.updateUri(openFileDialog.FileName);
             }
             //txtEditor.Text = File.ReadAllText(openFileDialog.FileName);
-        } 
+        }
+        private void Window_Closing(object sender, CancelEventArgs e)
```

Comment:
add line between functions

Reply:
and remove  the _

Reply:
See one function above - you are using it as well